static/app/views/settings/account/accountIdentities.spec.tsx

```diff
@@ -54,7 +55,9 @@ describe('AccountIdentities', function () {
 
     render(<AccountIdentities />);
 
-    expect(await screen.findByTestId('loading-indicator')).toBeInTheDocument();
+    await waitFor(() =>
```

Comment:
how did this statement go from `.toBeInTheDocument()` to be `.not.toBeInTheDocument()` ?

Reply:
shoot thats a good question

Reply:
oh wait, it's just a race condition between finding it in the document and the state updating to loaded